sqldelight-gradle-plugin/src/test/fixtures/well-formed-selects/expected/com/sample/HockeyPlayerModel.java

```diff
@@ -185,6 +185,50 @@
   @NonNull
   HockeyPlayer.Position position();
 
+  interface Select_allModel {
+    HockeyPlayerModel hockey_player();
+
+    TeamModel team();
+  }
+
+  interface For_teamModel {
+    HockeyPlayerModel hockey_player();
+
+    TeamModel team();
+  }
+
+  interface Subquery_joinModel {
+    long _id();
+
+    int age();
+  }
+
+  interface Select_expressionModel {
+    String first_name();
+
+    long count();
+  }
+
+  interface Expression_subqueryModel {
+    HockeyPlayerModel hockey_player();
+
+    long size();
+  }
+
+  interface Some_joinModel {
+    HockeyPlayerModel hockey_player();
+
+    TeamModel team();
+  }
+
+  interface With_queryModel {
```

Comment:
agreed, although I should mention that we originally had query names as camelCase but switched them to snake_case for the string constants. So we either have to change the `CaseFormat` or have the string constants be like

```java
interface TestModel {
  String withQuery = "SELECT * ...";
}
```